### PR TITLE
Russian translation of Devise views and errors

### PR DESCRIPTION
**Pull request by AlexSukov**

### CODE REVIEW

**Comment by AlexSukov** [comment]
@Magons russian translation of all. I think ^_^

File: app/views/devise/sessions/new.html.haml [hunk 1]
```diff
@@ -1,15 +1,19 @@
+- flash.each do |name, msg|
+  = content_tag :div, msg, :id => "flash_#{name}" if msg.is_a?(String)
```
Comment (AlexSukov) [inline app/views/devise/sessions/new.html.haml]: @vakyl9r @Magons IMPORTANT! There is no error message on Sign In page without this code!

File: app/views/devise/sessions/new.html.haml [hunk 2]
```diff
@@ -1,15 +1,19 @@
+- flash.each do |name, msg|
+  = content_tag :div, msg, id: "flash_#{name}" if msg.is_a?(String)
```
Comment (AlexSukov) [inline app/views/devise/sessions/new.html.haml:2]: @vakyl9r @Magons IMPORTANT! There is no error message on Sign In page without this code!

**Comment by Magons** [comment]
@AlexSukov if u start using local do it for every string. Call me I explain. And again use single quotes

**Pull request merged**